erpcgen/src/templates/c_coders.template

```diff
@@ -34,15 +34,17 @@ uint32_t {$info.sizeTemp};
 {% endif %}
 uint8_t * {$info.dataTemp};
 codec->readBinary(&{$info.sizeTemp}, &{$info.dataTemp});
+if({$info.sizeTemp})
```

Comment:
- if statements should have one space char between "if" and "(" 
- the added 2 lines should be moved 3 lines bellow (attaching the expected content)
[c_coders.zip](https://github.com/EmbeddedRPC/erpc/files/5022760/c_coders.zip)